### ISSUE TITLE
Exception if GPX does not contain line breaks

### ISSUE THREAD

**Issue by NopMap** [jenetics/jpx#85]
I was sent a GPX that could not be loaded by JPX 1.5.0. It throws the following exception.

> java.io.IOException: javax.xml.stream.XMLStreamException: Unexpected element <trkpt>.
> 	at io.jenetics.jpx.GPX$Reader.read(GPX.java:1143)
> 	at io.jenetics.jpx.GPX.read(GPX.java:1974)

Analysis showed that the format was ok, but with try-and-error I found that adding a single line break makes the file load without exception. Of course, XML should ignore any whitespace and line breaks around the tags.

Error can be reproduced with this GPX (no line breaks included):
```
<?xml version="1.0" encoding="UTF-8" standalone="no" ?><gpx xmlns="http://www.topografix.com/GPX/1/1" xmlns:gpxx="http://www.garmin.com/xmlschemas/GpxExtensions/v3" xmlns:wptx1="http://www.garmin.com/xmlschemas/WaypointExtension/v1" xmlns:gpxtpx="http://www.garmin.com/xmlschemas/TrackPointExtension/v1" creator="Oregon 450" version="1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd http://www.garmin.com/xmlschemas/GpxExtensions/v3 http://www8.garmin.com/xmlschemas/GpxExtensionsv3.xsd http://www.garmin.com/xmlschemas/WaypointExtension/v1 http://www8.garmin.com/xmlschemas/WaypointExtensionv1.xsd http://www.garmin.com/xmlschemas/TrackPointExtension/v1 http://www.garmin.com/xmlschemas/TrackPointExtensionv1.xsd"><metadata><link href="http://www.garmin.com"><text>Garmin International</text></link><time>2014-09-28T12:15:24Z</time></metadata><trk><name>28-SEP-14 14:15:22</name>
<extensions><gpxx:TrackExtension><gpxx:DisplayColor>Black</gpxx:DisplayColor></gpxx:TrackExtension></extensions><trkseg><trkpt lat="46.7155065201" lon="12.3633751553"><ele>1999.20</ele><time>2014-09-28T09:00:29Z</time></trkpt><trkpt lat="46.7155107111" lon="12.3633777536"><ele>1999.20</ele><time>2014-09-28T09:00:30Z</time></trkpt><trkpt lat="46.7155086156" lon="12.3633698747"><ele>1999.20</ele><time>2014-09-28T09:00:51Z</time></trkpt><trkpt lat="46.7155669536" lon="12.3635026440"><ele>1999.20</ele><time>2014-09-28T09:01:11Z</time></trkpt><trkpt lat="46.7155936919" lon="12.3635734711"><ele>1999.20</ele><time>2014-09-28T09:01:20Z</time></trkpt><trkpt lat="46.7158015631" lon="12.3637087550"><ele>2000.16</ele><time>2014-09-28T09:01:34Z</time></trkpt><trkpt lat="46.7157668620" lon="12.3638706096"><ele>2002.08</ele><time>2014-09-28T09:01:50Z</time></trkpt></trkseg></trk></gpx>
```
Adding a single line break after the closing tag `</extensions>` will make it load.

**Comment by jenetics** [comment]
This is the same issue as in #82 and has been fixed in [r1.5.1](https://github.com/jenetics/jpx/releases/tag/v1.5.1).

**Comment by NopMap** [comment]
I can confirm it is fixed in 1.5.1
